code/game/objects/items/defibrillator.dm

```diff
@@ -43,32 +49,33 @@
 	return ..()
 
 
-/obj/item/defibrillator/update_icon_state()
+/obj/item/defibrillator/update_icon()
```

Comment:
You kind of missed what I was saying here. Icon state changes should go in an update_icon_state, overlay changes should go in an update_overlays, and neither of those two should be called directly.